File: src/test/java/pages/MainMenuValidation.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class MainMenuValidation {
    public static void main(String[] args) {
        System.setProperty("webdriver.chrome.driver", "path/to/chromedriver.exe");
        WebDriver driver = new ChromeDriver();
        driver.get("https://www.orangehrm.com/");
        WebElement mainMenu = driver.findElement(By.id("MainMenu")); 
        String[] optionsToValidate = {
            "Admin",
            "PIM",
            "Leave",
            "Time",
            "Recruitment",
            "My Info",
            "Performance",
            "Dashboard",
            "Directory",
            "Maintenance",
            "Buzz"
        };

        for (String option : optionsToValidate) {
            String optionXPath = "//ul[@id='mainMenu']//a[contains(text(), '" + option + "')]";
            WebElement optionElement = mainMenu.findElement(By.xpath(optionXPath));
            if (optionElement.isDisplayed()) {
                System.out.println(option + " is displayed.");
            } else {
                System.out.println(option + " is not displayed.");
            }
        }

        // Close the WebDriver
        driver.quit();
    }
}

File: src/test/python/pages/MainMenuValidation.py
from selenium import webdriver

options_to_validate = [
    "Admin",
    "PIM",
    "Leave",
    "Time",
    "Recruitment",
    "My Info",
    "Performance",
    "Dashboard",
    "Directory",
    "Maintenance",
    "Buzz"
]

chrome_driver_path = "path/to/chromedriver.exe"
driver = webdriver.Chrome(chrome_driver_path)

driver.get("https://www.orangehrm.com/")
main_menu = driver.find_element_by_id("MainMenu")

for option in options_to_validate:
    option_xpath = f"//ul[@id='mainMenu']//a[contains(text(), '{option}')]"
    option_element = main_menu.find_element_by_xpath(option_xpath)
    if option_element.is_displayed():
        print(f"{option} is displayed.")
    else:
        print(f"{option} is not displayed.")

# Close the WebDriver
driver.quit()
